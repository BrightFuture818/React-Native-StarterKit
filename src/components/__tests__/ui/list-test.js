/**
 * Test to check if the component renders correctly
 */
/* global it expect */

import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import List from '@ui/List';

/**
 * Check if list renders correcly
 * and asserting it to the matching snapshot
 */
it('List renders correcly', () => {
  const tree = renderer.create(
    <List />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
