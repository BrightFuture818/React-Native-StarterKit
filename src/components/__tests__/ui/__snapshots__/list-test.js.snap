exports[`test List renders correcly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#ffffff",
        "borderBottomWidth": 1,
        "borderColor": "#cbd2d9",
        "borderTopWidth": 1,
        "marginTop": 20,
      },
      Array [
        Object {
          "backgroundColor": "#E9EBEE",
          "borderBottomWidth": 0,
          "borderTopColor": "#D0D1D5",
          "margin": 0,
        },
        Array [],
      ],
    ]
  } />
`;
